src/modules/RA/Viewpoint/components/ViewpointFields.js

```diff
@@ -146,19 +156,22 @@ const ViewpointFields = ({ edit, classes, ...props }) => {
 
         <FileField source="related" src="document" title="key" />
       </FormTab>
+      )}
 
+      {edit && (
       <FormTab label="resources.viewpoint.tabs.pictures" path="pictures">
         <ReferenceArrayField source="picture_ids" reference={RES_PICTURE} fullWidth>
           <Datagrid rowClick="edit">
-            <TextField source="owner" />
+            {false && <ReferenceField source="owner_id" reference="users" />}
```

Comment:
Wut?

Reply:
In addition to commit title, I added a code comment to explain it.